README.md

```diff
@@ -152,6 +153,50 @@ then once every 24 hours.
 Telemetry can be disabled by setting the metadata key `disable-guest-telemetry`
 to `true`.
 
+#### MTLS MDS
+
+GCE [Shielded VMs](https://cloud.google.com/compute/shielded-vm/docs/shielded-vm)
+now support HTTPS endpoint `https://metadata.google.internal/computeMetadata/v1`
+for Metadata Server. To enable communication with secure HTTPS endpoint, Guest Agent 
+retrieves and stores credentials on the VM's disk in a standard location, making them 
+accessible to any client application running on the VM. Both the root certificate 
+and client credentials are updated each time the guest-agent process starts. 
+Furthermore, client credentials are also automatically rotated every 48 hours 
```

Comment:
I think you should make it clear that "rotation" here means the guest-agent rotating client certificates on disk, and that the actual certificate validity lifetime is different from this (maybe mention that it's a bit longer so there's overlap) and let the cloud.google.com documentation specify certificate lifetime.

Reply:
Done, public docs not mention lifetime right now but we have a internal doc bug to update it with these details.